lib/src/protobuf/field_set.dart

```diff
@@ -129,7 +129,7 @@ class _FieldSet {
 
   List<T> _getDefaultList<T>(FieldInfo<T> fi) {
     assert(fi.isRepeated);
-    if (_isReadOnly) return const [];
+    if (_isReadOnly) return new List.unmodifiable(<T>[]);
```

Comment:
Alternative:
  return new List<T>.unmodifiable(const <Null>[]);

Reply:
Yep, that works. Thanks!